sdk/react/src/use_picovoice.ts

```diff
@@ -68,38 +60,31 @@ export function usePicovoice(
   const porcupineCallback = useRef(keywordCallback);
   const rhinoCallback = useRef(inferenceCallback);
 
-  const start = (): boolean => {
-    if (webVoiceProcessor !== null) {
-      webVoiceProcessor.start().then(() => {
-        setIsListening(true);
-        return true;
-      });
-    }
-    return false;
-  };
+  const wakeWordCallback = useCallback(
+    (detection: PorcupineDetection): void => {},
+    []
+  );
 
-  const pause = (): boolean => {
-    if (webVoiceProcessor !== null) {
-      webVoiceProcessor.pause();
-      setIsListening(false);
-      return true;
-    }
-    return false;
-  };
+  const inferenceCallback = useCallback((inference: RhinoInference): void => {},
+  []);
 
-  const stop = (): boolean => {
-    if (webVoiceProcessor !== null) {
-      webVoiceProcessor.stop().then(() => {
-        setIsListening(false);
-        setEngine('ppn');
-        if (picovoiceWorker !== null) {
-          picovoiceWorker.postMessage({ command: 'reset' });
-        }
-        return true;
-      });
-    }
-    return false;
-  };
+  const init = useCallback(
+    async (
+      accessKey: string,
+      keyword: PorcupineKeyword,
+      porcupineModel: PorcupineModel,
+      context: RhinoContext,
+      rhinoModel: RhinoModel,
+      options: PicovoiceOptions = {}
+    ): Promise<void> => {},
+    [wakeWordCallback, inferenceCallback]
```

Comment:
should we add all of them as deps?

Reply:
They aren't deps, but arguments into the callback so it's okay.